packages/mobile/src/fiatExchanges/FiatExchangeOptions.tsx

```diff
@@ -82,7 +83,7 @@ function CurrencyRadioItem({
           { borderColor: currencyBorderColor(selected) },
         ]}
       >
-        <RadioIcon selected={selected} />
+        <RadioButton selected={selected} />
```

Comment:
Shouldn't this also accept a `disabled` property, similar to how you've done below?

Reply:
thanks! forgot about dis one :P